wiki/Legacy_osu!store_items/en.md

```diff
@@ -0,0 +1,124 @@
+---
+tags:
+  - store
+  - merchandise
+  - legacy
+---
+
+# Past osu!store items
+
+This article lists various items that were once available at the [osu!store](https://osu.ppy.sh/store/listing) but are no longer in production or for sale.
+
+## osu! goods
+
+![osu! goods CafePress store front](img/osu-goods.png)
+
+The **osu! goods** were a collection of official osu!-branded merchandise sold through the online retailer [CafePress](https://en.wikipedia.org/wiki/CafePress) from 2009 – 2012. All items were printed and produced by CafePress, with a portion of the sales being used to [support the development of the game](https://osu.ppy.sh/store/listing).
+
+The collection featured a range of products such as:
+
+### Shirts
+
+- [osu! Mod Shirt](https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume.289786473)
+- [osu! Lite T-Shirt](https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume.288837174)
+- [Cookie Munch T-Shirt](https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume.288838261)
+- [osu! Girly T-Shirt](https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume.288831390)
+- [Organic Men's Fitted T-Shirt](https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume.439576382)
```

Comment:
Link here and some others don't seem to be archived. Or is it intentional that it may magically be archived?

Reply:
hmm i just copied everything here from the main store page on the internet archive : https://web.archive.org/web/20120702093250/http://www.cafepress.com/osume

i feel like some links aren't available simply because the internet archive simply didn't choose to archive them for some reason, and unfortunately there's nothing i can do here :(